File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
a, b, c = map(int,input().split())

if (a == b and a == c) :
    print(10000 + a * 1000)
    
elif (a == b and a != c) : 
    s = a
    print(1000 + s * 100)
elif (b == c and a != b) :
    s = b
    print(1000 + s * 100)
elif (a == c and a != b) :
    s = a
    print(1000 + s * 100)

else :
    print(max(a,b,c)* 100)

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.cc
#include <iostream>

using namespace std;

int main() {
	int a, b, c, max = 0;

	cin >> a >> b >> c;
	max = a;

	// 같은 눈 3개
	if ((a == b) && (a == c)) {
		cout << 10000 + a * 1000;
	}
	// 같은 눈 2개
	else if ((a == b) && (b != c)) {
		cout << 1000 + a * 100;
	}
	else if ((a != b) && (b == c)) {
		cout << 1000 + b * 100;
	}
	else if ((a != b) && (a == c)) {
		cout << 1000 + c * 100;
	}
	// 다 다르면?
	else {
		if (max < b) max = b;
		if (max < c) max = c;
		cout << max * 100;
	}

	return 0;
}